cmd/sdkch/README.md

```diff
@@ -0,0 +1,174 @@
+# sdkch
+Simple tool to maintain modular changelogs
```

Comment:
The flow of this documentation should be restructured to move from a basic users needs (aka only needs to know the add command) to the needs of somebody making a release (needs to know generate, as well as the structure etc).

Reply:
Ok

Reply:
I've revisited README, please review